core/rawdb/freezer_table.go

```diff
@@ -89,10 +89,8 @@ func (start *indexEntry) bounds(end *indexEntry) (startOffset, endOffset, fileId
 // It consists of a data file (snappy encoded arbitrary data blobs) and an indexEntry
 // file (uncompressed 64 bit indices into the data file).
 type freezerTable struct {
-	// WARNING: The `items` field is accessed atomically. On 32 bit platforms, only
-	// 64-bit aligned fields can be atomic. The struct is guaranteed to be so aligned,
-	// so take advantage of that (https://golang.org/pkg/sync/atomic/#pkg-note-BUG).
-	items uint64 // Number of items stored in the table (including items removed from tail)
+	items      atomic.Uint64 // Number of items stored in the table (including items removed from tail)
+	itemOffset atomic.Uint64 // Number of items removed from the table
```

Comment:
Why do we change itemOffset from 32-bit to 64-bit?

Reply:
As I see that type cast itemOffset to uint64 frequently happens and uint64 might be good for x64 machine when handle atomic read/write